crates/hyperqueue/src/client/output/json.rs

```diff
@@ -152,6 +152,8 @@ impl Output for JsonOutput {
         self.print(format_tasks(job.tasks, map));
     }
 
+    fn print_jobs_tasks(&self, _jobs: Vec<(JobId, JobDetail)>, _worker_map: WorkerMap) {}
```

Comment:
I think that we could easily implement this. Let's create a dictionary keyed by job ID and use `format_tasks` for the values.